packages/smooth_ui_library/lib/widgets/smooth_card.dart

```diff
@@ -27,20 +25,8 @@ class SmoothCard extends StatelessWidget {
             padding: const EdgeInsets.all(12.0),
             child: Column(
               children: <Widget>[
-                if (collapsed != null)
-                  Row(
-                    mainAxisSize: MainAxisSize.max,
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: <Widget>[
-                      Container(child: header),
-                      if (collapsed != null)
-                        Icon(collapsed
-                            ? Icons.keyboard_arrow_down
-                            : Icons.keyboard_arrow_up),
-                    ],
-                  ),
-                if (collapsed != true) content,
+                header,
```

Comment:
@M123-dev : I think this is causing #270 .  header should be required, or we should test if it is null. The AttributeListExpandable does not set it.